docs/userstyle-creation.md

```diff
@@ -84,7 +84,8 @@ on the schema:
    ```yaml
    <port>:
      name: <name>
-     category: <category>
+     # Up to 3 categories are allowed, where the left most category is the primary category.
```

Comment:
```suggestion
     # Up to 3 categories are allowed. Keep in mind that the first category is considered the "primary" category.
```

Reply:
I went with left most, since first is could be diffrent depending on what is your primary language, i.e. Arabic reading right to left, as to avoid confusion.